package.json

```diff
@@ -3,6 +3,10 @@
   "displayName": "ctest-lab",
   "description": "CTest extension for Visual Studio Code",
   "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brobeson/ctest-lab.git"
+  },
```

Comment:
`vsce` was complaining that there is not repository information in the _package.json_. It also complains about the lack of a license but I didn't know which license you wanted to put this under.

Reply:
```c++
"licence": "MIT"
```

Thanks for pointing this out.